platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,742 @@
+
+To use Seqera Pipelines to import the `nf-core-rnaseq` pipeline to your workspace:
+
+![Seqera Pipelines add to Launchpad](./_images/pipelines-add.gif)
+
+1. Search for `nf-core-rnaseq` and select **Launch** next to the pipeline name in the list. In the **Add pipeline** tab, select **Cloud** or **Enterprise** depending on your Platform account type, then provide the information needed for Seqera Pipelines to access your Platform instance:
+    - **Seqera Cloud**: Paste your Platform **Access token** and select **Next**.  
+    - **Seqera Enterprise**: Specify the **Seqera Platform URL** (hostname) and **Base API URL** for your Enterprise instance, then paste your Platform **Access token** and select **Next**.
+    :::tip
+    If you do not have a Platform access token, select **Get your access token from Seqera Platform** to open the Access tokens page in a new browser tab.
+    :::
+1. Select your Platform **Organization**, **Workspace**, and **Compute environment** for the imported pipeline. 
+1. (Optional) Customize the **Pipeline Name** and **Pipeline Description**.
+1. Select **Add Pipeline**. 
+
+:::info
+To add a custom pipeline not listed in Seqera Pipelines to your Platform workspace, see [Add pipelines](./quickstart-demo/add-pipelines.mdx#) for manual Launchpad instructions. 
+:::
+
+## Pipeline input data
+
+The [nf-core/rnaseq](https://github.com/nf-core/rnaseq) pipeline works with input datasets (samplesheets) containing sample names, FASTQ file locations, and indications of strandedness. 
+
+<details>
+  <summary>**Example nf-core/rnaseq dataset**</summary>
+    
+  | sample              | fastq_1                              | fastq_2                              | strandedness |
+  | ------------------- | ------------------------------------ | ------------------------------------ | ------------ |
+  | WT_REP1             | s3://nf-core-awsmegatests/rnaseq/... | s3://nf-core-awsmegatests/rnaseq/... | reverse      |
+  | WT_REP1             | s3://nf-core-awsmegatests/rnaseq/... | s3://nf-core-awsmegatests/rnaseq/... | reverse      |
+  | WT_REP2             | s3://nf-core-awsmegatests/rnaseq/... | s3://nf-core-awsmegatests/rnaseq/... | reverse      |
+  | RAP1_UNINDUCED_REP1 | s3://nf-core-awsmegatests/rnaseq/... |                                      | reverse      |
+  | RAP1_UNINDUCED_REP2 | s3://nf-core-awsmegatests/rnaseq/... |                                      | reverse      |
+  | RAP1_UNINDUCED_REP2 | s3://nf-core-awsmegatests/rnaseq/... |                                      | reverse      |
+  | RAP1_IAA_30M_REP1   | s3://nf-core-awsmegatests/rnaseq/... | s3://nf-core-awsmegatests/rnaseq/... | reverse      |
+
+</details>
```

Comment:
I would also actually suggest that since we are talking about the `test_full` dataset here, why not show that samplesheet with the paired fastq instead?
https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/v3.10/samplesheet_full.csv